### DISCUSSION TITLE
requirements.yml for docker install

### DISCUSSION THREAD

**Discussion by zenxedo** [semaphoreui/semaphore#1559] [Q&A]
I don't know where to place the requirements.yml in the docker install. The docker compose file has only one volume for the config file.

**Comment by hmoffatt** [comment]
The requirements.yml for Ansible Galaxy? I think Semaphore is expecting it to be in your repository with the playbooks.

**Comment by SvartrShell** [comment]
did you ever suss this? I cant find anything online and have same issue - tried putting the requirements file in my repo root, in a folder called collections - still never detects it
